codegen/smithy-aws-typescript-codegen/src/test/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfigTest.java

```diff
@@ -31,15 +33,127 @@ public void addsAwsRuntimeConfigSettings() {
                 .build();
         new TypeScriptCodegenPlugin().execute(context);
 
-        // Check that one of the many dependencies was added.
+        // Check dependencies
         assertThat(manifest.getFileString("package.json").get(),
-                   containsString(AwsDependency.CREDENTIAL_PROVIDER_NODE.packageName));
```

Comment:
Some of this logic was moved to AddAwsAuthPlugin a while back, but the tests weren't updated. I've moved the auth specific tests to AddAwsAuthPluginTest